packages/metal-dom/src/domData.js

```diff
@@ -10,20 +10,12 @@ class domData {
 	 * Gets Metal.js's data for the given element.
 	 * @param {!Element} element
 	 * @param {string=} name Optional property from the data to be returned.
-	 * @param {*=} initialValue Optional value to the set the requested property
-	 *     to if it doesn't exist yet in the data.
 	 * @return {!Object}
 	 */
-	static get(element, name, initialValue) {
```

Comment:
Hey @diegonvs,

I don't think we should remove any functionality from the `get` method, since that will be a breaking change (even if we account for our own use cases it might break somewhere else).

Can we maintain the `initialValue` argument, and maybe just call the new `set` method from inside `get` if the arg is passed?

Reply:
good idea Rob!

Reply:
Good point @Robert-Frampton , i think in the `get` method there could be no creation or editing of domData but thinking so as not to break API, this makes sense.

Reply:
Since we're leaving the API alone for `get`, we should also leave all the instances of it alone in metal. For example, https://github.com/metal/metal.js/pull/317/commits/41756bc8e4d8ca01b99058bbcbaa770125fe4ed5 should just be left using `get`.